doc/user_guide/user_guide.md

```diff
@@ -188,6 +188,17 @@ final VirtualSchema virtualSchema = factory.createVirtualSchemaBuilder("THE_VIRT
         .build();
 ```
 
+### Running SQL From Files to Create Objects
+
+Implementation often come with SQL files that users need to execute as a preparation. Since those files contain production code, it needs to be tested &mdash; but first you need to run those SQL scripts.
```

Comment:
```suggestion
Implementation often comes with SQL files that users need to execute as a preparation. Since those files contain production code, it needs to be tested &mdash; but first you need to run those SQL scripts.
```